src/js/tabs/firmware_flasher.js

```diff
@@ -480,20 +549,6 @@ TABS.firmware_flasher.initialize = function (callback) {
             }).change();
         });
 
-        chrome.storage.local.get('show_development_releases', function (result) {
-            if (result.show_development_releases) {
-                $('input.show_development_releases').prop('checked', true);
-            } else {
-                $('input.show_development_releases').prop('checked', false);
-            }
-
-            self.releaseChecker.loadReleaseData(buildBoardOptions);
```

Comment:
This one should be reintroduced to poll for new build info when entering the tab